### ISSUE TITLE
Teachingbubble missing required ARIA attribute: aria-level

### ISSUE THREAD

**Issue by SummerCat200** [microsoft/fluentui#13441]
### Environment Information

- **Package version(s)**: 7.107.6
- **Browser and OS versions**: Chrome

### Describe the issue:

Run fastpass for accessibility insights for web extension on teachingBubble page, it shows teachingBubble headline missing aria attribute: aria-level, and I couldn't find a way to pass it to teachingBubble  

#### Actual behavior:

![Capture](https://user-images.githubusercontent.com/52942564/83573091-a5876e80-a4df-11ea-8fc1-a90ac984f9ea.PNG)

**Comment by jdhuntington** [comment]
Thanks for reporting @SummerCat200 -- I believe you're correct that this attribute cannot be 
passed. Are you willing to submit a pull request?

I can think of a few ways this might be improved, but picking a proper header element instead of a `p` might be the best option. `src/components/TeachingBubble/TeachingBubbleContent.base.tsx` line 102 seems to be the place to actually wire it in, though not sure how exposed the value should be.
